src/commands/Suggestions/suggest.ts

```diff
@@ -0,0 +1,93 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+import { MessageEmbed, TextChannel } from 'discord.js';
+import { SkyraCommand } from '@lib/structures/SkyraCommand';
+import { GuildSettings } from '@lib/types/settings/GuildSettings';
+import { BrandingColors } from '@utils/constants';
+import { displayEmoji } from '@utils/util';
+import { PermissionLevels } from '@lib/types/Enums';
+
+export default class extends SkyraCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			cooldown: 10,
+			description: language => language.tget('COMMAND_SUGGEST_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_SUGGEST_EXTENDED'),
+			requiredPermissions: ['EMBED_LINKS'],
+			runIn: ['text'],
+			usage: '<suggestion:string{3,2000}>'
+		});
+	}
+
+	public async run(message: KlasaMessage, [suggestion]: [string]) {
+		let suggestionsChannelID: string | null = message.guild!.settings.get(GuildSettings.Suggestions.SuggestionsChannel);
+
+		// If we don't have a suggestions channel setup, initiate setting it up
+		if (!suggestionsChannelID) {
+			const userResponse = await this.setChannel(message);
+			if (!userResponse) return;
+			suggestionsChannelID = userResponse;
+		}
+
+		const suggestionsChannel = this.client.channels.get(suggestionsChannelID) as TextChannel;
+		// Get the next suggestion ID
+		const suggestionID = message.guild!.settings.get(GuildSettings.Suggestions.AscendingID);
+
+		// Post the suggestion
+		const suggestionsMessage = await suggestionsChannel!.send(new MessageEmbed()
+			.setColor(BrandingColors.Primary)
+			.setAuthor(`${message.author.tag} (${message.author.id})`, message.author.displayAvatarURL({ size: 128 })!)
+			.setTitle(message.language.tget('COMMAND_SUGGEST_TITLE', suggestionID))
+			.setDescription(suggestion));
+
+		// Add the upvote/downvote reactions
+		const reactArray = await message.guild!.settings.pluck(GuildSettings.Suggestions.VotingEmojis.UpvoteEmoji, GuildSettings.Suggestions.VotingEmojis.DownvoteEmoji) as string[];
+		for (const emojiStr of reactArray) {
+			const emoji = displayEmoji(emojiStr);
```

Comment:
That's broken code, emojis are serialized in the exact form the API accepts them.